index.html

```diff
@@ -97,11 +97,17 @@ <h1 class="multi-post-title">Multi-Post Stories</h1>
             <li class="multi-btn">Bootstrap</li>
             <li class="multi-btn">Ruby</li>
           </ul>
-          <button class="see-project-btn for-hover for-pressed" type="button">See Project</button>
+          <a><button
+            class="see-project-btn for-hover for-pressed click-popup"
+            data-position="0"
+            type="button"
+          >
+            See Project
+          </button></a>
```

Comment:
- [ ] Youn have put a great effort on this milestone ,but to make this your website look great and professional you have not implement the changes requested by the previouse reviewer [see here](https://github.com/mryeminthu/Portfolio/pull/22#discussion_r1238000251) 
- [ ] According to the project requirement, all of your works card should be generated dynmaically using JavaScript rather than throgh html like you are currently doing. To achieve this you would make use of the projects array you have created and dynamically loop through its contents to display them on the webpage when the page loads and remove all the pieces of the html code after you have make it dynamic on you javascript so that you code will be easy to read .

Reply:
@ademibowale  Hello Sir,
I think my repo display dynamically. Could you please kindly tell me in detail what I should remove from html.